maven-publish.gradle

```diff
@@ -0,0 +1,73 @@
+apply plugin: "maven-publish"
+apply plugin: "signing"
+
+// Dokka is optional but recommended.
+// Sonatype will not accept a published artifact without at least an empty Javadoc JAR.
+task javadocsJar(type: Jar, dependsOn: "dokkaJavadoc") {
+    getArchiveClassifier().set("javadoc")
+    from dokkaJavadoc.outputDirectory
+}
+
+task sourcesJar(type: Jar) {
+    getArchiveClassifier().set("sources")
+    from sourceSets.main.allJava
+}
+
+afterEvaluate {
+    publishing {
+        // gets specific props defined per each module which breaks the need for multiple(per module) maven-publish files
+        def properties = project.ext.mavenPublishProperties
+
+        repositories {
+            maven {
+                name "SonatypeMavenCentral"
+                url properties.repository.url
+                credentials {
+                    username properties.repository.username
+                    password properties.repository.password
+                }
+            }
+        }
+        publications {
+            release(MavenPublication) {
+                // Only needed for non Android modules. Uncomment if it is needed.
+//                from components.java
+
+                groupId = properties.group
+                version = properties.version
+                artifactId = properties.artifactId
+
+//                artifact bundleReleaseAar
+                artifact sourcesJar
+                artifact javadocsJar
+
+                pom {
+                    name = properties.name
+                    description = properties.description
+                    url = properties.url
+                    licenses {
+                        license {
+                            name = 'The Apache License, Version 2.0'
+                            url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                        }
+                    }
+                    developers {
+                        developer {
+                            id = 'malicombo'
+                            name = 'Mali Čombo'
```

Comment:
What is even Čombo 😂

Reply:
A Čovjek with a big head :D

Reply:
@JonatanPlesko @schalke95 Let's also discuss if we're going to have one company wide developer (android@infinum.com) on all the libraries or everyone would update this list whenever they made a change.
Afaik this isn't connected to any account, just for informational purposes.